### PR TITLE
fuel guage is working

### PR DESCRIPTION
**Pull request by Mangesh55**
My fuel gauge is working...
guage

### CODE REVIEW

File: game.js [hunk 1]
```diff
@@ -19,13 +20,27 @@ function initGame() {
...
 function loop() {
   
     if (thrust) {
+        if(fuel>0){
+            fuel-=1;
+            displayFuel(fuel);
+
+                    
+        }
+        if(fuel==0){
```
Comment (tcroft) [inline game.js:42]: This isn't right, when the fuel goes to 0 that just means you can't make the ship fly anymore, eventually it'll drop and crash.  Crashing here is premature

File: index.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
             <div><button onclick="reset()">Reset</button></div>
             <div><button onmousedown="startThrust()" onmouseup="stopThrust()">Thrust</button></div>
             <div id="fuel">fuel</div>
-            
+            <input type="range" id="a" value="50">
```
Comment (tcroft) [inline index.html]: This widget doesn't seem to do anything.  If you are looking for an interesting and simple way to make a gauge, take a look at the html meter widget

File: game.js (hunk 1 above)
Comment (Mangesh55) [reply game.js:42]: ok I'll do it , and thanks for confirming..

File: index.html (hunk 2 above)
Comment (Mangesh55) [reply index.html]: ok ..

File: game.js (hunk 1 above)
Comment (Mangesh55) [reply game.js:42]: fuelguage

**Pull request closed without merging**